core/templates/pages/exploration-editor-page/modal-templates/save-validation-fail-modal.component.spec.ts

```diff
@@ -54,9 +54,9 @@ class MockWindowRef {
         if (this._hashChange === null) {
           return;
         }
-        this._hashChange();
+        this._hashChange;
```

Comment:
Why is this changed?

Reply:
_hasChange is not a function, it's a variable. It's useless so removed it now.